apps/api/src/app/organization/usecases/create-organization/create-organization.command.ts

```diff
@@ -14,7 +14,7 @@ export class CreateOrganizationCommand extends AuthenticatedCommand {
   public readonly logo?: string;
 
   @IsOptional()
-  @IsEnum(JobTitleEnum)
+  @IsString()
```

Comment:
What was the purpose of this change? If it's a trouble with typings, we should coerce the type instead.